common/src/main/java/com/wynntils/core/features/overlays/OverlayPosition.java

```diff
@@ -86,35 +86,35 @@ public String toString() {
 
     public static OverlayPosition getBestPositionFor(
             Overlay overlay, float oldRenderX, float oldRenderY, float offsetX, float offsetY) {
-        Vec2 topLeftOfOverlay = new Vec2(overlay.getRenderX(), overlay.getRenderY());
+        Vec2 referencePoint = new Vec2(
```

Comment:
This reference point is the center of the overlay, yes? Do you think it would be helpful in the GUI to render it, so the user can see which section the center is inside? Or will it just be confusing?